dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/managegroups/ManageGroupsCRUDServiceIT.java

```diff
@@ -0,0 +1,122 @@
+import edu.harvard.iq.dataverse.authorization.AuthenticationServiceBean;
+import edu.harvard.iq.dataverse.authorization.groups.impl.explicit.ExplicitGroupServiceBean;
+import edu.harvard.iq.dataverse.persistence.dataverse.Dataverse;
+import edu.harvard.iq.dataverse.persistence.group.ExplicitGroup;
+import edu.harvard.iq.dataverse.persistence.user.RoleAssignee;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.transaction.api.annotation.TransactionMode;
+import org.jboss.arquillian.transaction.api.annotation.Transactional;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.ejb.EJB;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+@Transactional(TransactionMode.ROLLBACK)
+@RunWith(Arquillian.class)
+public class ManageGroupsCRUDServiceIT extends WebappArquillianDeployment {
+
+    @PersistenceContext(unitName = "VDCNet-ejbPU")
+    private EntityManager em;
+
+    @EJB
+    private ManageGroupsCRUDService manageGroupsCRUDService;
+    @EJB
+    private DataverseServiceBean dataverseService;
+    @Inject
+    private DataverseSession dataverseSession;
+    @EJB
+    private AuthenticationServiceBean authenticationService;
+    @EJB
+    private ExplicitGroupServiceBean explicitGroupService;
+
+    @Before
+    public void setUp() {
+        dataverseSession.setUser(authenticationService.getAdminUser());
+    }
+
+    @Test
+    public void shouldCreateGroup() {
+        // given
+        Dataverse dv = dataverseService.findByAlias("ownmetadatablocks");
+        RoleAssignee roleAssignee = authenticationService.findAllAuthenticatedUsers().get(0);
+
+        // when
+        manageGroupsCRUDService.create(dv, "testGroup", "testGroupId", "testDesc", Collections.singletonList(roleAssignee));
+
+        // then
+        List<ExplicitGroup> dbExplicitGroupList = explicitGroupService.findByOwner(dv.getId());
+        Assert.assertTrue(dbExplicitGroupList
+                .stream()
+                .anyMatch(explicitGroup -> explicitGroup.getDisplayName().equals("testGroup")));
+        Assert.assertTrue(dbExplicitGroupList
+                .stream()
+                .filter(explicitGroup -> explicitGroup.getDisplayName().equals("testGroup"))
+                .findFirst().get().structuralContains(roleAssignee));
+    }
+
+    @Test
+    public void shouldUpdateGroup() {
+        // given
+        Dataverse dv = dataverseService.findByAlias("ownmetadatablocks");
+        ExplicitGroup explicitGroup = getExplicitGroup(dv);
+
+        em.persist(explicitGroup);
+
+        // when
+        explicitGroup.setDisplayName("testGroup");
+        manageGroupsCRUDService.update(explicitGroup, new LinkedList<>());
```

Comment:
If we would add roleassignee instead of empty list here, then test would be more complete. (we should also assert role assignees in then section